tests/contrib/django_legacy/test_tracing_disabled.py

```diff
@@ -0,0 +1,41 @@
+# 3rd party
+from django.test import TestCase
+
+# project
+from ddtrace.tracer import Tracer
+from ddtrace.contrib.django.conf import settings
+from ddtrace.contrib.django.legacy import patch as _ready
```

Comment:
Left over from me: we can leave the import as `patch` no need to call it `_ready`